can/broadcastmanager.py

```diff
@@ -275,16 +275,14 @@ def stop(self) -> None:
 
     def start(self) -> None:
         self.stopped = False
+        if USE_WINDOWS_EVENTS:
+            win32event.SetWaitableTimer(
+                self.event.handle, 0, self.period_ms, None, None, False
+            )
         if self.thread is None or not self.thread.is_alive():
             name = f"Cyclic send task for 0x{self.messages[0].arbitration_id:X}"
             self.thread = threading.Thread(target=self._run, name=name)
             self.thread.daemon = True
-
-            if USE_WINDOWS_EVENTS:
```

Comment:
Wasn't `SetWaitableTimer` called anyway? `self.thread.is_alive()` should return False on a restart, right?